src/server/server.js

```diff
@@ -80,6 +83,15 @@ app.get('/health', (req, res) => {
   res.status(OK).json({ status: OK, text: 'Health check ok' });
 });
 
+app.post('/format-html', (req, res) => {
+  const html = prettier.format(req.body.html, {
+    parser: 'html',
+    printWidth: 1000000, // Avoid inserting linebreaks for long inline texts i.e. <p>Lorem ......... ipsum</p>
+    plugins: [parseHTML],
```

Comment:
Er det mulig å bruke htmlWhitespaceSensitivity: 'strict' her, for å ikke endre whitespace ved formatering? Da kunne man kanskje kjørt format som default også

Reply:
Ville fortsatt ha problemer med whitspace inne i <p>. Det ville sett slik ut: 
<img width="577" alt="screenshot 2019-02-08 at 10 37 06" src="https://user-images.githubusercontent.com/364853/52470111-931c4f80-2b8d-11e9-9048-e880995581ed.png">